internal/services/formatters/csharp/scs/formatter.go

```diff
@@ -54,24 +54,30 @@ func (f *Formatter) StartAnalysis(projectSubPath string) {
 		return
 	}
 
-	f.SetAnalysisError(f.startSecurityCodeScan(projectSubPath), tools.SecurityCodeScan, projectSubPath)
+	output, err := f.startSecurityCodeScan(projectSubPath)
+	f.SetAnalysisError(err, tools.SecurityCodeScan, output, projectSubPath)
 	f.LogDebugWithReplace(messages.MsgDebugToolFinishAnalysis, tools.SecurityCodeScan, languages.CSharp)
 }
 
-func (f *Formatter) startSecurityCodeScan(projectSubPath string) error {
+//nolint:funlen
+func (f *Formatter) startSecurityCodeScan(projectSubPath string) (string, error) {
 	f.LogDebugWithReplace(messages.MsgDebugToolStartAnalysis, tools.SecurityCodeScan, languages.CSharp)
 
 	analysisData := f.getDockerConfig(projectSubPath)
 	if err := f.verifyIsSolutionError(analysisData.CMD); err != nil {
-		return err
+		return analysisData.CMD, err
```

Comment:
Why return the `analysisData.CMD` with an error? I think that we should return an empty string when some error occur.

Reply:
was added an commentary about this:
```
* Another error found was that the error logs presented by the tool were scattered, so we centralized the SetAnalysisError function where, in addition to adding the error in the analysis, it also performs the unified log among all the tool
```